src/test/java/org/sonarsource/scanner/cli/LogsTest.java

```diff
@@ -22,16 +22,18 @@
 import java.io.PrintStream;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
 import org.mockito.ArgumentMatchers;
 import org.mockito.Mock;
-import org.mockito.MockitoAnnotations;
+import org.mockito.junit.jupiter.MockitoExtension;
 import org.sonarsource.scanner.lib.LogOutput;
 
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.reset;
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.verifyNoMoreInteractions;
 
+@ExtendWith(MockitoExtension.class)
```

Comment:
I know there is probably no consensus on this, but I tend to not use `MockitoExtension`.
I think writing:
```
private final Foo foo = mock();
```
is better than
```
@Mock
private Foo foo;
```